lib/std/os/bits.zig

```diff
@@ -34,3 +34,22 @@ pub const iovec_const = extern struct {
     iov_base: [*]const u8,
     iov_len: usize,
 };
+
+// syslog
+
+/// system is unusable
+pub const LOG_EMERG = 0;
```

Comment:
These should be an enum.
`std.log.Level` might be reusable.

Reply:
I don't want to couple `std.log.Level` with this.

Reply:
The `std.log.Level` options intentionally match the syslog protocol.

Reply:
```c
syslog(LOG_ERR|LOG_USER, "error");
```